packages/api/exec.mts

```diff
@@ -84,7 +82,18 @@ export function tsc(options: NodeRequestType) {
   return spawnCall({
     command: Path.join(cwd, 'node_modules', '.bin', 'tsc'),
     cwd,
-    args: [entry, ...tsConfigToArgs(DEFAULT_TSCONFIG)],
+    args: [
```

Comment:
Hardcoding for now. This is going to go away once the tsserver pr is merged anyways.